src/components/serviceRequest/ServiceInformationSection.tsx

```diff
@@ -97,7 +97,7 @@ export default function ServiceInformationSection(props: ServiceInformationSecti
             <div className="grid col-12">
               <div className="col-fixed mr-3">{serviceInformationLabels.Source}</div>
               <div className="flex flex-wrap gap-3">
-                {sources.map((opt) => (
+                {sources ? sources.map((opt) => (
```

Comment:
Why was this made optional as well?

Reply:
Not optional, just a conditional check to see if sources are defined. Its possible they are null or undefined (with this old version of useAppConstants)